config/config.go

```diff
@@ -283,6 +284,7 @@ func loadDefaultServicesConfig(config *SSIServiceConfig) {
 		},
 		WebhookConfig: WebhookServiceConfig{
 			BaseServiceConfig: &BaseServiceConfig{Name: "webhook"},
+			WebhookTimeout:    "10s",
```

Comment:
should this be a `time.Duration` value in go parsed from a string in config?

Reply:
I think keeping it a string would be more consistent to the rest of the code and not lock you into using one duration framework, also because of the shortcomings of toml I would have to make a messy wrapper and custom logic to parse a string to a time.Duration object.